Makefile

```diff
@@ -48,7 +48,10 @@ build-container-library: init
 
 install-container-library: init
 	# temporarily bypass urllib3 because circular dependency will be introduced if bumped up urllib3 version
-	pipenv run safety check -i 43975 # https://github.com/pyupio/safety
+	# temporarily bypass py=1.1.0 because pytest-parallel has a dependency on it however the module is no longer maitained. 
+	# In the future the pylib will be removed from pytest-parallel dependency and 51457 should only impact the local tests.
+	# For more info, https://github.com/pytest-dev/py/issues/287
+	pipenv run safety check -i 43975 -i 51457 # https://github.com/pyupio/safety
```

Comment:
The ignore is recommended by the author. So temp to approve this to unblock Can's MCM.